docs: Refactor of Build with AI section (#2044)
apify/apify-docs: sources/platform/actors/development/quick_start/build_with_ai.md

---
title: Build with AI
sidebar_position: 3
description: Use pre-built prompts, refer to Apify Docs via llms.txt, and follow best practices for effective vibe coding.
slug: /actors/development/quick-start/build-with-ai
toc_max_heading_level: 4
---

> **Use pre-built prompts, reference Apify Docs via llms.txt, and follow best practices to build Actors efficiently with AI coding assistants.**

Reviewer: ```suggestion
**Use pre-built prompts, reference Apify docs via llms.txt, and follow best practices to build Actors efficiently with AI coding assistants.**
```

---
title: Build with AI
sidebar_position: 3

> description: Use pre-built prompts, use Apify docs via llms.txt, and follow best practices to effective vibe coding.

Reviewer: ```suggestion
description: Use pre-built prompts, refer to Apify Docs via llms.txt, and follow best practices for effective vibe coding.
```

slug: /actors/development/quick-start/build-with-ai
toc_max_heading_level: 4
---

> **Use pre-built prompts, use Apify docs via llms.txt, and follow best practices to build Actors efficiently with AI assistants.**

Reviewer: ```suggestion
**Use pre-built prompts, reference Apify Docs via llms.txt, and follow best practices to build Actors efficiently with AI coding assistants.**
```

Reviewer: The same here: use .... use ....

---

import { AGENTS_PROMPT } from "@site/src/utils/agents-prompt";
import PromptButton from "@site/src/components/PromptButton";
import InstallMCPButton from "@site/src/components/InstallMCPButton";
import copyForAI from "./images/copy-for-ai.png";

This guide shows you how to build Actors efficiently with AI coding assistants. You'll learn how to use pre-built instructions, integrate Apify documentation into your AI editor, and apply best practices for AI-assisted development.

## AI coding assistant instructions

Use the following prompt in your AI coding assistant ([Cursor](https://www.cursor.com/), [Claude Code](https://www.claude.com/product/claude-code), [GitHub Copilot](https://github.com/features/copilot), etc.):

<PromptButton prompt={AGENTS_PROMPT} title="Use pre-built prompt for your AI coding assistant" />

### Quick Start

- _Step 1_: Create directory: `mkdir my-new-actor`
- _Step 2_: Open the directory in _Cursor_, _Claude Code_, _VS Code with GitHub Copilot_, etc.
- _Step 3_: Copy the prompt above and paste it into your AI coding assistant (Agent or Chat)

> - _Step 4_: Run it, and develop your first actor with the help of AI 🎉

Reviewer 2: But how does that differ than using 

1.
2. 
3. 
4. 

This is already established pattern in documentation, while using `Step 1` feel out of place

:::info Start in an AI coding assistant

To maximize efficiency, copy the prompt to Cursor, VS Code with GitHub Copilot, or another AI coding assistant. The AI will follow the guide step-by-step, and you'll avoid copy-pasting from tools like ChatGPT or Claude.

:::

## Use Actor templates with AGENTS.md

> All [Actor Templates](https://apify.com/templates) have AGENTS.md that will help you with AI coding. If you prefer a more traditional approach, run the following command:

Reviewer: ```suggestion
If you’d rather start with an existing [Actor template](https://apify.com/templates)
, use the included AGENTS.md file (bundled with every template) to get started with AI-augmented development.
Create a new template with the following command:

```
